app/src/main/java/io/github/droidkaigi/confsched2017/view/customview/OverScrollLayout.java

```diff
@@ -138,11 +132,11 @@ public void onAnimationRepeat(Animator animation) {
                     });
             return;
         }
-        if (getY() != originalLocation.y) {
+        if (getY() != originalRect.top) {
```

Comment:
I have a question. 🙋 
In here's statement, is it needless to check `overScrollListener !=null`?
I think this animation is to restore `overscroll` animation above, so I'm wondering this is really needed if `overScrollListener == null`.